pandas/_libs/lib.pyx

```diff
@@ -107,7 +109,10 @@ def memory_usage_of_objects(arr: object[:]) -> int64_t:
 def is_scalar(val: object) -> bint:
     """
     Return True if given value is scalar.
-
+    
+    Parameters
+    ----------
+    val : input argument of any type
```

Comment:
After the name of the parameter and the colon, just but the name of the expected Python type. In this case use `object` if anything can be used.